pom.xml

```diff
@@ -160,6 +161,7 @@
           </execution>
         </executions>
       </plugin>
+      -->
```

Comment:
Deploying to the maven central requires gpg singing, while deploying to snapshot repository doesn't. For now, we deploy only to the snapshot repository. I'll think about a better way to handle this later.